.env.example

```
@@ -5,3 +5,12 @@ STRAVA_CLIENT_ID=your_client_id_here
 STRAVA_CLIENT_SECRET=your_client_secret_here
 STRAVA_CLIENT_ACCESS_TOKEN=your_access_token_here
 STRAVA_REDIRECT_URI=http://localhost:3000/api/strava/callback
+
+# Database Configuration
+# PostgreSQL connection settings
+
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_DB=regionriders
+POSTGRES_USER=regionriders_user
+POSTGRES_PASSWORD=regionriders_pass
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)